courses/tests/test_api.py

```diff
@@ -72,9 +78,48 @@ def test_create_course_accepts_instructors(self):
                 "961e87a0803e436b989cb62d5e672c5f"
             ]
         })
-        self.assertEqual(resp.status_code, 201)
+        self.assertEqual(resp.status_code, 201, resp.content)
```

Comment:
nit: since we do not use the optional `msg` , maybe call it using kwarg (but I leave this up to you)